Tasks/SshV0/ssh.ts

```diff
@@ -91,39 +90,39 @@ async function run() {
             tl.setResult(tl.TaskResult.Failed, tl.loc('ConnectionFailed', err));
         }
 
-        if(sshClientConnection) {
+        if (sshClientConnection) {
             //SSH connection successful
             tl._writeLine(tl.loc('SshConnectionSuccessful'));
             if (runOptions === 'commands') {
                 //run commands specified by the user
-                for (var i:number = 0; i < commands.length; i++) {
-                    tl.debug('Running command ' + commands[i] + ' on remote machine.');
-                    tl._writeLine(commands[i]);
-                    var returnCode:string = await sshHelper.runCommandOnRemoteMachine(
-                        commands[i], sshClientConnection, remoteCmdOptions);
-                    tl.debug('Command ' + commands[i] + ' completed with return code = ' + returnCode);
+                for (const command of commands) {
+                    tl.debug('Running command ' + command + ' on remote machine.');
+                    tl._writeLine(command);
+                    const returnCode: string = await sshHelper.runCommandOnRemoteMachine(
+                        command, sshClientConnection, remoteCmdOptions);
+                    tl.debug('Command ' + commands + ' completed with return code = ' + returnCode);
                 }
             } else { // both other runOptions: inline and script
                 //setup script path on remote machine relative to user's $HOME directory
-                var remoteScript = './' + path.basename(scriptFile);
-                var remoteScriptPath = '"' + remoteScript + '"';
-                var windowsEncodedRemoteScriptPath = remoteScriptPath;
-                var isWin = os.type().match(/^Win/);
+                const remoteScript = './' + path.basename(scriptFile);
+                let remoteScriptPath = '"' + remoteScript + '"';
+                const windowsEncodedRemoteScriptPath = remoteScriptPath;
+                const isWin = os.platform() === 'win32';
```

Comment:
This returns one of the following strings:

![image](https://user-images.githubusercontent.com/33549821/50027753-5253f000-ffbb-11e8-8c37-2f8d46b9490e.png)